core/src/main/groovy/noe/server/Tomcat.groovy

```diff
@@ -668,16 +670,21 @@ class Tomcat extends ServerAbstract implements WorkerServer {
   void enableSslOpenSsl() {
     def nl = platform.nl
     def dummyComment = '<!-- Define an AJP 1.3 Connector on port 8009 -->'
-    def enableOpenSsl = '<Connector port="' + this.mainHttpsPort.toString() + '" SSLEnabled="true"' + nl +
-        'maxThreads="200" scheme="https" secure="true"' + nl +
-        'SSLCertificateFile="' + this.sslCertificate + '"' + nl +
-        'SSLCertificateKeyFile="' + this.sslKey + '"' + nl +
-        'SSLPassword="' + this.sslKeystorePassword + '"' + nl
-    if(platform.isFips()) {
-      enableOpenSsl += 'ciphers="' + DefaultProperties.FIPS_140_2_CIPHERS + '"'
+    def ciphers = ''
+    if (platform.isFips()) {
+      ciphers = 'ciphers="' + DefaultProperties.FIPS_140_2_CIPHERS + '"'
     }
-    enableOpenSsl += '/>'
-
+    def enableOpenSsl =
+            '<Connector port="' + this.mainHttpsPort.toString() + '" SSLEnabled="true"' + nl +
```

Comment:
@sgala it might be worth explicitly declaring the protocol here too `protocol="org.apache.coyote.http11.Http11NioProtocol"`

Reply:
It was tested with both tomcat 9 and tomcat 10.1 

Re: protocol, I'm not sure it is worth it, as we only care about the ssl implementation in this "enableOpenssl..." methods